include/ua_client_highlevel.h

```diff
@@ -210,6 +212,44 @@ UA_Client_readUserExecutableAttribute(UA_Client *client, const UA_NodeId nodeId,
                                      &UA_TYPES[UA_TYPES_BOOLEAN]);
 }
 
+/**
+ * Historical Access
+ * ^^^^^^^^^^^^^^^^^
+ * The following functions can be used to read a single node historically.
+ * Use the regular service to read several nodes at once.
+ */
+#ifdef UA_ENABLE_HISTORIZING
+typedef UA_Boolean
+(*UA_HistoricalIteratorCallback)(UA_Client *client,
+                                 const UA_NodeId *nodeId,
+                                 UA_Boolean moreDataAvailable,
+                                 const UA_ExtensionObject *data, void *callbackContext);
+
+#ifdef UA_ENABLE_EXPERIMENTAL_HISTORIZING
+UA_StatusCode UA_EXPORT
+UA_Client_HistoryRead_events(UA_Client *client, const UA_NodeId *nodeId,
```

Comment:
We normally use a lower case letter after the underscore. I.e. this should be `UA_Client_historyRead_events` to be consistent.

Also for the other definitions below.

Reply:
Nope. In the client we have like "modules" that are CamlCase starting upper-case.
The first verb starts lowercase...

For example `UA_Client_MonitoredItems_modify`

Reply:
Ah, right, sorry, then that's correct :+1: